docs/getting-started/building.md

```diff
@@ -52,10 +52,14 @@ This guide will demonstrate how to build a Zarhus OS image from zero!
 
 ## Build
 
+Depending on which features you want to have in your build, pass the desired
+`.yml` files via command line. You can read more on that in
+[kas documentation](https://kas.readthedocs.io/en/latest/userguide/project-configuration.html#including-configuration-files-via-the-command-line)
+
 From `yocto` directory run:
 
 ```shell
-$ SHELL=/bin/bash kas-container build meta-zarhus/kas-IMAGE_TYPE.yml
+SHELL=/bin/bash kas-container build meta-zarhus/kas/common.yml:meta-zarhus/kas/rockchip.yml
 ```
 
 !!! note
```

Comment:
You should delete this note.

Reply:
[Addressed](https://github.com/zarhus/docs/pull/7/files#diff-5977c316ebed8f6694cc31e4303e065a23c0faeecf63c9a9795e617a66a9d7e6L61-L64)